### PR TITLE
MINOR FIX : Student Attachment Service Exception Add

### PR DESCRIPTION
**Pull request by tharanisantosh**

### CODE REVIEW

File: OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.StudentAttachments.Exceptions
+{
+    public class FailedStudentAttachmentServiceException : Xeption
+    {
+        public FailedStudentAttachmentServiceException(Exception innerException)
+            : base(message: "Failed student attachment service error occured,contact support." ,innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs:14]: : base(message: "Failed student attachment service error occurred, contact support.", innerException)

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.StudentAttachments.Exceptions
+{
+    public class FailedStudentAttachmentServiceException : Xeption
+    {
+        public FailedStudentAttachmentServiceException(Exception innerException)
+            : base(message: "Failed student attachment service error occured,contact support.",innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs:14]: : base(message: "Failed student attachment service error occurred, contact support.", innerException)

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs (hunk 1 above)
Comment (tharanisantosh) [reply OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs]: Comment Resolved. Thanks Humrudha.

File: OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs [hunk 3]
```diff
@@ -11,7 +11,7 @@ namespace OtripleS.Web.Api.Models.StudentAttachments.Exceptions
     public class FailedStudentAttachmentServiceException : Xeption
     {
         public FailedStudentAttachmentServiceException(Exception innerException)
-            : base(message: "Failed student attachment service error occured,contact support.",innerException)
+            : base(message:"Failed student attachment service error occured,contact support.",innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs]: : base(message: "Failed student attachment service error occurred, contact support.", innerException)

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.StudentAttachments.Exceptions
+{
+    public class FailedStudentAttachmentServiceException : Xeption
+    {
+        public FailedStudentAttachmentServiceException(Exception innerException)
+            :base(message:"Failed student attachment service error occured,contact support.",innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/StudentAttachments/Exceptions/FailedStudentAttachmentServiceException.cs:14]: : base(message: "Failed student attachment service error occurred, contact support.", innerException)

Please take care of the alignment

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**